src/components/Dropdown/dropdown.css

```diff
@@ -2,8 +2,55 @@
 @import "src/css/variables/fonts.css";
 @import "src/css/variables/colors.css";
 
-.y-dropdown .ms-Dropdown-label{
+.y-dropdown .ms-Dropdown-label {
   @extend text__mediumSub;
   font-weight: $fontWeight__bold;
   color: $textColor__secondary;
 }
+
+.y-dropdown .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown-caretDownWrapper {
+  height: 36px;
+  line-height: 3.3rem;
+}
+
+.y-dropdown .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .y-dropdown__caretDown {
+  color: $color__river;
+}
+
+.y-dropdown .ms-Dropdown:hover .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown:focus .ms-Dropdown-title {
+  color: $textColor;
+  border-color: $borderColor;
+}
+
+.y-dropdown .ms-Dropdown:active .ms-Dropdown-title {
+  border-color: $color__lake;
```

Comment:
Why not name this too?

Reply:
@mamuso looking at mockups from you and Kike, the focused Textfield's border color is our link color (river, #386cbb) while the focused Dropdown border color is lake (#6c98d9). Should these be consistent?

@bryan-gislason i'd recommend making a new variable in colors.css like `$focusedBorderColor` set to $color__lake. if @mamuso says focused input borders should be our link color then i'd still keep $focusedBorderColor as a separate variable and set its value to $linkColor

Reply:
and @mamuso has verified that the zeplin mocks are our source of truth, so in colors.css:

`$focusedBorderColor: $color__lake;`

Reply:
should this be named something more like: borderColor--focused ?

Reply:
ah good call